ftw/faqblock/profiles/default-plone5/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+    <dependencies>
+        <dependency>profile-ftw.upgrade:default</dependency>
```

Comment:
Don't we need ftw.simplelayout for this block?

Reply:
Nice catch this was already wrong from the start 👍